udkm1Dsim/structures/layers.py

```diff
@@ -884,8 +888,8 @@ def __str__(self):
                               self.atoms[i][2],
                               '',
                               self.atoms[i][0].mag_amplitude,
-                              self.atoms[i][0].mag_phi.magnitude,
-                              self.atoms[i][0].mag_gamma.magnitude,
+                              self.atoms[i][0].mag_phi.magnitude.to('deg').magnitude,
```

Comment:
I think this should read  `self.atoms[i][0].mag_phi.to('deg').magnitude`, i.e., without the `.magnitude` in front of the `.to`. Otherwise the print in the UnitCell example of the documentation  fails

Reply:
thanks